devimint/src/tests.rs

```diff
@@ -1818,6 +1818,117 @@ pub async fn cannot_replay_tx_test(dev_fed: DevFed) -> Result<()> {
     Ok(())
 }
 
+pub async fn recurringd_test(dev_fed: DevFed) -> Result<()> {
```

Comment:
nit: I would probably prefer this to be in the `recurringd` and use devimint as a library